helm/prometheus-rules/templates/platform/atlas/alerting-rules/grafana.rules.yml

```diff
@@ -23,3 +23,110 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    {{- if eq .Values.managementCluster.provider.flavor "vintage" }}
```

Comment:
Can you remove this?

Reply:
Sorry something went wrong during the merging with main. Fixed :heavy_check_mark: